src/azure-cli-core/azure/cli/core/util.py

```diff
@@ -735,16 +735,17 @@ def send_raw_request(cli_ctx, method, url, headers=None, uri_parameters=None,  #
     # Replace common tokens with real values. It is for smooth experience if users copy and paste the url from
     # Azure Rest API doc
     from azure.cli.core._profile import Profile
-    profile = Profile()
+    profile = Profile(cli_ctx=cli_ctx)
```

Comment:
Pass the existing `cli_ctx` to `Profile`. Otherwise, `Profile` will create a new `AzCli` which is unnecessary. 

https://github.com/Azure/azure-cli/blob/fcc93da55f8b360618f7afe880cb2f106c9ae0e9/src/azure-cli-core/azure/cli/core/_profile.py#L152